sdk/loadtestservice/azure-developer-loadtest/README.md

```diff
@@ -0,0 +1,161 @@
+provide an instance of the desired credential type obtained from the
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+
+Use the returned token credential to authenticate the client:
+
+```python
+from azure.developer.loadtesting import LoadTestClient
+from azure.identity import DefaultAzureCredential
+client = LoadTestClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+```
+
+## Examples
+
+### Creating a load test 
+```python
+from azure.developer.loadtesting import LoadTestClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+client = LoadTestClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
+try:
+    result = client.load_test_administration.create_or_update_test(
+        TEST_ID,
+        {
+            "resourceId": f"/subscriptions/{SUBSCRIPTION_ID}/resourceGroups/yashika-rg/providers/Microsoft.LoadTestService/loadtests/loadtestsdk",
+            "description": "",
+            "displayName": DISPLAY_NAME,
+            "loadTestConfig": {
+                "engineSize": "m",
+                "engineInstances": 1,
+                "splitAllCSVs": False,
+            },
+            "secrets": {},
+            "environmentVariables": {},
+            "passFailCriteria": {"passFailMetrics": {}},
+            "keyvaultReferenceIdentityType": "SystemAssigned",
+            "keyvaultReferenceIdentityId": None,
+        },
+    )
+    print(result)
+except HttpResponseError as e:
+     print('service responds error: {}'.format(e.response.json()))
+
+```
+
+### Uploading .jmx file to a Test
+```python
+try:
+    # opening .jmx file
+    body = {}
+    body["file"] = open("sample.jmx", "rb")
+
+    result = client.load_test_administration.upload_test_file(TEST_ID, FILE_ID, body)
+    print(result)
+except HttpResponseError as e:
+    print("Failed to send JSON message: {}".format(e.response.json()))
+```
+
+### Running a Test
+```python
+try:
+    result = client.load_test_runs.create_and_update_test(
+        TEST_RUN_ID,
+        {
+            "testId": TEST_ID,
+            "displayName": DISPLAY_NAME,
+            "requestSamplers": [],
+            "errors": [],
+            "percentiles": ["90"],
+            "groupByInterval": "5s",
+        },
+    )
+    print(result)
+except HttpResponseError as e:
+    print("Failed to send JSON message: {}".format(e.response.json()))
```

Comment:
it is same for other variables (`TEST_RUN_ID`, `TEST_ID`, etc): https://github.com/Azure/azure-sdk-for-python/pull/25264/files#r927257552

Reply:
Resolved in https://github.com/Azure/azure-sdk-for-python/pull/25264/commits/faddc05c876a7926ac822e27fce970217dab4929